ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -208,12 +205,22 @@ open class BadgeField: UIView {
         isAccessibilityElement = false
     }
 
+    @objc private func themeDidChange(_ notification: Notification) {
+        guard let window = window, window.isEqual(notification.object) else {
```

Comment:
nit: since you won't need the non-option window reference, couldn't you do just: 
`guard window?.isEqual(notification.object)` ?